src/Microsoft.DncEng.PatGeneratorTool/Program.cs

```diff
@@ -1,22 +1,24 @@
 using Azure.Core;
 using Azure.Identity;
 using Microsoft.DncEng.PatGenerator;
-using Microsoft.IdentityModel.Clients.ActiveDirectory;
-using Microsoft.VisualStudio.Services.Client;
 using Microsoft.VisualStudio.Services.Common;
 using System;
 using System.Collections.Generic;
 using System.CommandLine;
 using System.CommandLine.Invocation;
 using System.Linq;
+using System.Security;
 using System.Text;
 using System.Threading.Tasks;
+using Microsoft.Identity.Client;
+using Microsoft.VisualStudio.Services.OAuth;
 
 namespace Microsoft.DncEng.PatGeneratorTool
 {
     class Program
     {
         private const int DefaultExpirationInDays = 30;
+        private const string AzdoClientId = "872cd9fa-d31f-45e0-9eab-6e460a02d1f1";
```

Comment:
What is this value?  I'm curious where it comes from/what it's representing.

Reply:
This came from this Azdo sample:

https://github.com/microsoft/azure-devops-dotnet-samples/blob/main/ClientLibrary/Quickstarts/dotnet/MaterializeUserQuickStarts/Program.cs#L18

Reply:
I'm a little worried we are just copying GUID's we dont' understand. My _guess_ is that this is some application ID that the samples are using and we should probably have our own application registered. But that's hard to tell when it's just a random GUID with no context.

We should probably provide feedback on the sample that they really need to explain what this value is and what we are supposed to pass in it's place.

Reply:
It is the Visual Studio app client ID.

Reply:
Ah. It doesn't seem like we should be claiming to be the visual studio application, because we aren't the visual studio application.  We should _ideally_ try and be our own application.